src/sentry/integrations/vsts/integration.py

```diff
@@ -463,8 +468,36 @@ def build_integration(self, state: Mapping[str, Any]) -> Mapping[str, Any]:
             integration_model = IntegrationModel.objects.get(
                 provider="vsts", external_id=account["accountId"], status=ObjectStatus.ACTIVE
             )
-            # preserve previously created subscription information
-            integration["metadata"]["subscription"] = integration_model.metadata["subscription"]
+
+            # Get Integration Metadata
+            integration_migrated = integration_model.metadata.get("integration_migrated", False)
+            if (
+                features.has(
+                    "organizations:migrate-azure-devops-integration", self.pipeline.organization.id
+                )
+                and not integration_migrated
+            ):
+                subscription_id, subscription_secret = self.create_subscription(
```

Comment:
Just because I'm unfamiliar with this integration: Is this for creating a new webhook subscription, or something else?

Reply:
what this does is setup the client and subscribe to the webhook to sync from devops -> sentry
https://github.com/getsentry/sentry/blob/1548e7a18767d982633ee648695abd3c2413208a/src/sentry/integrations/vsts/client.py#L119-L134